projects/bp-gallery/environments/.env.production

```diff
@@ -3,3 +3,5 @@ VITE_REACT_APP_CONTACT_MAIL=archiv@bad-harzburg-stiftung.de
 VITE_REACT_APP_SENTRY_DSN=https://315a3c261a1c4b5d81de2241b7e7cfc9@o1243479.ingest.sentry.io/6458997
 VITE_REACT_APP_ADVANCED_SEARCH=false
 VITE_REACT_APP_MATOMO_URL=//matomo.bad-harzburg-stiftung.de/
+VITE_REACT_APP_GROWTHBOOK_APIHOST=https://growthbook.harz-history.de/api
+VITE_REACT_APP_GROWTHBOOK_CLIENTKEY=sdk-tv4J1V8S0dYt4kOy
```

Comment:
I think this is fine, because the value will be visible in the devtools either way, but just to be sure: This isn't a secret, i. e. something we should not publish in the repo?

Reply:
Yep not secret. Just part of the setup for growthbook - same as sentry, matomo urls only here with api host and client key